### PR TITLE
Added integration tests for api's

### PR DESCRIPTION
**Pull request by Dimundra**

### CODE REVIEW

File: server/controllers/formController.js [hunk 1]
```diff
@@ -19,6 +44,7 @@ const getFormByIdHandler = async (request, h) => {
       err.stack
     );
   });
+  console.log(form);
```
Comment (kibernetika) [inline server/controllers/formController.js:47]: remove this console

File: server/package.json [hunk 2]
```diff
@@ -4,8 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "launch-frontend": "cd ../client && npm run build && cd ../server && node index.js",
-    "test": "jest --detectOpenHandles"
+    "migrate-all-db": "npx sequelize db:migrate && cross-env NODE_ENV=test npx sequelize db:migrate",
```
Comment (kibernetika) [inline server/package.json:7]: the word 'all' is redundant in the command name

File: server/package.json [hunk 3]
```diff
@@ -4,8 +4,10 @@
...
+    "seed-all-db": "npx sequelize db:seed:all && cross-env NODE_ENV=test npx sequelize db:seed:all",
```
Comment (kibernetika) [inline server/package.json:8]: the same as above

**Review by kibernetika — APPROVED** [review]

File: server/controllers/formController.js (hunk 1 above)
Comment (Dimundra) [reply server/controllers/formController.js:47]: Done!

File: server/package.json (hunk 2 above)
Comment (Dimundra) [reply server/package.json:7]: Done!

File: server/package.json (hunk 3 above)
Comment (Dimundra) [reply server/package.json:8]: Done!

**Pull request merged**